backend/services/interfaces/classService.ts

```diff
@@ -0,0 +1,53 @@
+import { UserDTO, Grade } from "../../types";
+import { TestSessionResponseDTO } from "./testSessionService";
+
+export interface ClassRequestDTO {
+  className: string;
+  schoolYear: number;
+  gradeLevel: Grade[];
+  teacher: string;
+  testSessions: string[];
```

Comment:
we have added a `students: Student[];` field to the Class model! take a look at staging for the updates

Reply:
Hello, so I am not sure if I should include the `id: number;` in the interface here in line 4, but without it a bunch of errors pop up. Putting the id here means I have to initialize the id to something in `testStudents` in `backend/testUtils/class.ts`, which is kind of awkward since Mongo assigns its own id. Second thing, should id be a string in the class model instead of number?

Reply:
oops didn't catch that earlier - id should be a string! you should also have a 
- `StudentRequestDTO` has no id (used in ClassRequestDTO)
- `StudentResponseDTO` has id  (used in ClassReponseDTO)

Reply:
Ahh that makes a lot more sense. I shall make the id change, and also add `StudentRequestDTO` and `StudentResponseDTO` before the next work session, I got the wrong idea through our Slack communications thinking it wasn't needed LOL